easybuild/tools/asyncprocess.py

```diff
@@ -65,6 +65,8 @@
 :author: Jens Timmerman (Ghent University)
 """
 
+import fcntl  # @UnresolvedImport
+import select  # @UnresolvedImport
 import errno
 import os
```

Comment:
@Flamefire Please maintain alphabetical order after moving.

Also, are those `@UnresolvedImport` still needed? Those are modules from the Python std lib, so OK to remove imho.

```python
import errno
import fcntl
import os
import select
import subprocess
```